crates/bevy_input/src/touch.rs

```diff
@@ -0,0 +1,99 @@
+use bevy_app::{EventReader, Events};
+use bevy_ecs::{Local, Res, ResMut};
+use bevy_math::Vec2;
+use std::collections::{HashMap, HashSet};
```

Comment:
lets use the `bevy_utils::{HashMap, HashSet}` here. They're still the std types, but they use the much faster aHash algorithm instead of the defaults.